### PR TITLE
Dm quiz

### PR DESCRIPTION
**Pull request by Dmart331**
Fixed a few issues. Now meets requirements, just need to make less clunky.

### CODE REVIEW

File: quiz.js [hunk 1]
```diff
@@ -20,9 +22,26 @@ var treeStuff ={};
...
+    if (newRows.length > 1 || newChar.length > 1) {
+    	alert("ONLY ONE CHARACTER");
+    	    }
+     else if (event.keyCode == 13) {
+        document.getElementById("submit").click();
+    }
+
+    else if(event.keyCode < 48 || event.keyCode > 57 || event.keyCode != 8){
+        alert ("Only CHARACTERS no Letters");
+    }
```
Comment (JoeShep) [inline quiz.js]: I like that you're trying something different for form validation, but it's not working as you expect. If I use `tab` to jump from input to input it pops up the alert error. Also, hitting `shift` to type any character on the number keys does the same thing. It also won't work for hitting `enter` to submit the form inputs. It even threw an error when I used `command + tab` to jump from one browser to another when I had your app running : )

**Review by JoeShep — COMMENTED** [review]
Hey, Drew, you're nearly there. See my comment below about the bug in your form validation. Get that straightened out then you're good to go.

**Comment by JoeShep** [comment]
@Dmart331 I know there's a lot on your plate, but I'd like to see this get to 100% and merged.

File: quiz.js [hunk 2]
```diff
@@ -22,25 +23,52 @@ var char = document.getElementById("char");
...
 
+form.addEventListener("keyup", function(keyPress){
+	if(keyPress.keyCode === 13){
+		document.getElementById('submit').click();
+	}
+})
+
+
+
+
```
Comment (JoeShep) [inline quiz.js:74]: For future reference, make sure you don't leave empty lines at the end of your file. Best practice is a single blank line. There's a Sublime setting to assure that.

File: quiz.js [hunk 3]
```diff
@@ -22,25 +23,52 @@ var char = document.getElementById("char");
...
+}
+    else if (newRows.length > 1 || newChar.length > 1) {
+    	alert("ONLY ONE CHARACTER");
 
+	}
+	 else if ( event.keycode === 9){
+	 	alert("only one special char")
+	 }
+   
 });
```
Comment (JoeShep) [inline quiz.js:61]: So, you essentially have the same functionality repeated for `char` and `rows`. In the future, be aware of repeating yourself and look for ways to dry up your code.

**Review by JoeShep — APPROVED** [review]
Hey, Drew. This is working a lot better now. There are a couple of things I have noted below to watch out for in future projects, and there's still one odd bit of behavior you can look into if you have time: when I build a tree, if I immediately click or hit enter again, it builds two trees. 

Not a deal breaker at this point, since I'd like you concentrate on more immediate work. 

Keep up the good work; you're heading in the right direction

File: quiz.js (hunk 2 above)
Comment (Dmart331) [reply quiz.js:74]: Awesome, thanks Joe!

**Pull request merged**